### PR TITLE
it87.c: mostly formal code findings

### PR DESCRIPTION
**Pull request by airbjorn**
- fixing indents in #defines: replacing spaces with tabs, because some were mixed up with spaces and tabs differently from line to line
- variable assignment: replaced two spaces with one
- for loop: auto adjust threshold depending on array size

### CODE REVIEW

File: it87.c [hunk 1]
```diff
@@ -1982,7 +1982,7 @@ static ssize_t set_pwm_freq(struct device *dev, struct device_attribute *attr,
 	val *= has_newer_autopwm(data) ? 256 : 128;
 
 	/* Search for the nearest available frequency */
-	for (i = 0; i < 7; i++) {
+	for (i = 0; i < (ARRAY_SIZE(pwm_freq) - 1); i++) {
```
Comment (groeck) [inline it87.c]: outer () is unnecessary.

**Comment by groeck** [comment]
first two patches merged. Thanks!

File: it87.c (hunk 1 above)
Comment (airbjorn) [reply it87.c]: Thanks! Removed them with the latest commit within this request.

**Comment by groeck** [comment]
all merged.

**Pull request closed without merging**